packages/application-generic/src/services/calculate-delay/calculate-delay.service.spec.ts

```diff
@@ -6,31 +6,31 @@ describe('Calculate Delay Service', function () {
     const calculateDelayService = new CalculateDelayService();
 
     it('should convert seconds to milliseconds', function () {
-      const result = calculateDelayService.toMilliseconds(
+      const result = (calculateDelayService as any).toMilliseconds(
```

Comment:
The need of adding this `any` is a bit weird. What was the compiler complaining about?